edit-post/components/sidebar/post-schedule/index.js

```diff
@@ -3,31 +3,48 @@
  */
 import { __ } from '@wordpress/i18n';
 import { PanelRow, Dropdown, Button } from '@wordpress/components';
+import { withInstanceId } from '@wordpress/compose';
 import { PostSchedule as PostScheduleForm, PostScheduleLabel, PostScheduleCheck } from '@wordpress/editor';
+import { Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 
-export function PostSchedule() {
+export function PostSchedule( { instanceId } ) {
 	return (
 		<PostScheduleCheck>
 			<PanelRow className="edit-post-post-schedule">
-				<span>{ __( 'Publish' ) }</span>
+				<label
+					htmlFor={ `edit-post-post-schedule__toggle-${ instanceId }` }
+					id={ `edit-post-post-schedule__heading-${ instanceId }` }
```

Comment:
Why does the label need an `id`?

Reply:
It was like that when I got here. I missed it. Can probs remove. 👍